web/client/themes/default/less/cesium.less

```diff
@@ -28,4 +28,8 @@
     text-shadow: none;
     display: table-cell;
     vertical-align: middle;
-}
\ No newline at end of file
+}
+
+#map[data-sidebar] .cesium-viewer .compass {
```

Comment:
I would remove the [data-sidebar] and apply always the 40px
the results without sidebar and burgermenu as an acceptable margin
![image](https://user-images.githubusercontent.com/19175505/167130596-ea882fee-fd49-4d09-94cc-8f9cc8ec0d16.png)

Reply:
Reverted commit and added static position change regardless of sidebar availability